docker-compose.yml

```diff
@@ -41,11 +41,15 @@ services:
     networks:
       elonet:
         ipv4_address: 172.16.11.4
-  elo_adminer:
-    container_name: elo_adminer
-    image: adminer
+  elo_phpmyadmin:
+    container_name: elo_phpmyadmin
+    image: phpmyadmin/phpmyadmin
+    depends_on:
+      - elo_db
     ports:
-    - 127.0.0.1:$DB_ADMINER_PORT:8080
+      - 127.0.0.1:$DB_ADMINER_PORT:80
```

Comment:
można zmienić nazwę tej stałej, może na $PHPMYADMIN_PORT

Reply:
Racja, ale wtedy na jakąś bardziej uniwersalną.

Reply:
Myślę, że warto jednak tutaj skonkretyzować, że chodzi o phpMyAdmin, więc nie uogólniałem jej.

Chyba, że rzucisz pomysłem na lepszą nazwę, @ktrzos

Reply:
Dla mnie osobiście też może to być nazwa związana z phpMyAdminem, skoro tak nazywa się nasz docker (elo_phpmyadmin). Jeśli tu miałabybyć bardziej uniwersalna nazwa to również docker powinien mieć bardziej uniwersalną nazwę. Ale jak dla mnie skonkretyzowanie że chodzi o phpMyAdmina jest ok.